curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-destructuring-assignment-to-pass-an-object-as-a-functions-parameters.english.md

```diff
@@ -26,8 +26,7 @@ const profileUpdate = ({ name, age, nationality, location }) => {
 }
 ```
 
-This removes some extra lines and makes our code look neat.
-This has the added benefit of not having to manipulate an entire object in a function — only the fields that are needed are copied inside the function.
+When <code>profileData</code> is passed to the above function, the values are destructed from the function parameter for use within the function.
```

Comment:
```suggestion
When <code>profileData</code> is passed to the above function, the values are destructured from the function parameter for use within the function.
```

Reply:
@Manish-Giri Good catch!  In an effort to get as many curriculum PRs merged by the first of next week, would you mind pushing your suggestion as a commit?  It is just a typo, so that should be fine with everyone.

Reply:
@RandellDawson Done!